api/v1/policy_types.go

```diff
@@ -98,6 +102,9 @@ type PolicySpec struct {
 	// PolicyDependencies is a list of dependency objects detailed with extra considerations for
 	// compliance that should be fulfilled before applying the policies to the managed clusters.
 	Dependencies []PolicyDependency `json:"dependencies,omitempty"`
+
+	// HubTemplateOptions changes the default behavior of hub templates.
+	HubTemplateOptions *HubTemplateOptions `json:"hubTemplateOptions,omitempty"`
```

Comment:
This is a pointer so that the `omitempty` has effect rather than returning an empty struct when `hubTemplateOptions` is unset.

Reply:
Do we need to create HubTemplateOptions.ServiceAccountName? because it is only one property.

Reply:
We don't need it but I chose it in case we make other things customizable and I thought a `hubTemplateServiceAccountName` field was too long lol.

Reply:
Grouping more pretty and this is good for future maybe